bucket/juce.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://juce.com/",
+    "description": "JUCE is an open-source cross-platform C++ application framework. The leading framework for multi-platform audio applications",
+    "version": "5.4.7",
+    "license": {
+        "identifier": "JUCE Personal,JUCE Indie,JUCE Pro,JUCE Educational,ISC,GPL-3.0-or-later",
+        "url": "https://juce.com/juce-5-license"
+    },    
```

Comment:
just do
```
"license":"GPL-3.0-or-later",
```

Reply:
It actually has a multi tier license. OK I'll shorten